.github/workflows/run-tests.yml

```diff
@@ -368,10 +385,10 @@ jobs:
           TEST_BATCH: ${{matrix.test-batch}}
           QGIS_HTTPBIN_HOST: "localhost:8082"
         run: |
-          DOCKERFILE=$( ( [[ ${TEST_BATCH} == "ORACLE"   ]] && echo "docker-compose-testing-oracle.yml" ) \
-                     || ( [[ ${TEST_BATCH} == "POSTGRES" ]] && echo "docker-compose-testing-postgres.yml" ) \
+          DOCKERFILE=$( ( [[ ${{ matrix.test-batch }} == "ORACLE"   ]] && echo "docker-compose-testing-oracle.yml" ) \
+                     || ( [[ ${{ matrix.test-batch }} == "POSTGRES" ]] && echo "docker-compose-testing-postgres.yml" ) \
                      || echo "docker-compose-testing.yml" )
-          [[ ${TEST_BATCH} == "ORACLE" ]] && sudo rm -rf /usr/share/dotnet/sdk
+          [[ ${{ matrix.test-batch }} == "ORACLE" ]] && sudo rm -rf /usr/share/dotnet/sdk
```

Comment:
After https://github.com/actions/virtual-environments/issues/3338 this line doesn't make sense and can be deleted.

Reply:
can you maybe make a separate PR with this?
I doubt this PR will be merged anytime soon...